ls1intum/Artemis
# `Development`: Fix docker command in Jenkins development setup

### Checklist
#### General



### Motivation and Context

I followed all steps of the setup guide in the documentation, but docker did not work for me during the execution of jobs for programming exercises (I'm on mac). This PR fixes that by installing docker in the container instead of mounting the executable into it, as the socket is still mounted it is now possible to execute docker in the Artemis jenkins image.


It would be great if I could get a review from people using:
### Description


### Steps for Testing

Prerequisites:
- GitLab and Jenkins Dev Setup

1. Log in to Artemis
2. Create a programming exercise and check if the solution build is green, meaning that the docker image in the build plan was executed correctly.

### Testserver States
> [!NOTE]
> These badges show the state of the test servers.
> Green = Currently available, Red = Currently locked

![](https://byob.yarr.is/ls1intum/Artemis/artemis-test1)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test2)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test3)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test4)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test5)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test6)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test9)

No need to deploy it, it only affects/fixes your local setup.

### Review Progress






#### Performance Review
#### Code Review
#### Manual Tests

### Screenshots
